### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by ptamas88** [facebookresearch/UnsupervisedMT#15]
Hello,
Can you help me how to infer with the saved checkpoint?
I am stuck with loading back the model to torch and call model.eval() function.
Thanks

**Comment by glample** [comment]
Hi,

Currently the simplest way to do this would be to binarize the data you want to translate (being sure the data is also using the same BPE codes), and then reload the model with the `--eval_only 1` parameter, and specifying your binarized dataset as the test set. Then, the model will generate hypothesis for your dataset in the directory where the experiment is dumped. I'm sorry if this is not very convenient, I'll try to add a script that only takes as input the source file and the MT model later.

**Comment by akanshajainn** [comment]
Did you work on it? @glample 
I too want to use the trained model, but can't figure out how?
What would be the full command to use --eval_only parameter? How to provide input as a sentence?
What are the parameters we'll have to give?
